### ISSUE TITLE
Faster way to jump between cameras

### ISSUE THREAD

**Issue by F-Burning** [tahoma2d/tahoma2d#860]
Actually, to activate an specific camera you need to go to schematic, so, I would to switch between cameras in a easy and faster way, as for example from the Scene menu, or the timeline.

I've seen that you can do it from timeline, but when I select another camera nothing happens in the scene, although I could be wrong.

**Comment by artisteacher** [comment]
Yeah, it looks like switching cameras in the Timeline doesn't change which camera is activated. Personally, I think it should...
